nodejs/week1/warmUp/circleCalculator.js

```diff
@@ -0,0 +1,28 @@
+const x = Math.PI.toFixed(2);
+
+class Circle {
+
+	constructor(radius) {
+		this.radius = radius;
+    }
+    
+    getDiameter(){
+        const diameter = this.radius * 2;
+        console.log(diameter);
+    }
+
+    getCircumference(){
+       const circumference = this.radius * 2 * x;
+       console.log(circumference);
+    }
+
+    getArea(){
+         const area = this.radius * this.radius* x;
```

Comment:
This looks great 👍

You may want to sort the methods alphabetically. That can make them easier to find. This is not required but you may encounter that different companies have styleguides which such recommendations.